tests/test_item_tools.py

```
@@ -5,7 +5,7 @@
 """
 
 import pytest
-from unittest.mock import patch, MagicMock
+from unittest.mock import patch, MagicMock, AsyncMock
```

Ruff (0.12.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)